src/main/java/com/serhiiklymchuk/gmailpet/controller/UserController.java

```diff
@@ -0,0 +1,48 @@
+package com.serhiiklymchuk.gmailpet.controller;
+
+import com.serhiiklymchuk.gmailpet.dto.UserDto;
+import com.serhiiklymchuk.gmailpet.service.impl.UserServiceImpl;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping("/user")
+public class UserController {
+
+    private final UserServiceImpl userService;
+
+    public UserController(UserServiceImpl userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping("/home")
+    public String getHome(){
```

Comment:
Do you think `/home` should be under `UserController`?

Reply:
`/home` , `home.ftl` and everything which is linked with `/home` will be replaced with `/messages/inbox` in the next PR

Reply:
https://github.com/SerhiiKlymchuk/gmail-pet/pull/9#discussion_r560041103